robocorp-code/pyproject.toml

```diff
@@ -39,6 +39,8 @@ pillow = "^9.0"
 pywin32 = { version = ">=300,<304", platform = "win32", python = "!=3.8.1" }
 psutil = "^5.9.0"
 comtypes = "^1.1"
+java-access-bridge-wrapper = "^1.1.0"
```

Comment:
`java-access-bridge-wrapper` also needs to be added to the conda files in `/robocorp-code/bin/create_env/` (that's the actual production env, `pyproject.toml` is just for the dev env).